BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -0,0 +1,44 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include "Timer.h"
+
+
+int update_Time(int row)
+{
+    FILE *file = fopen("Timer.csv", "r");
+    char arr[256];
+
+    int counter=0;
+
+    int current;
+
+    int reload;
+
+
+    while(fgets(arr,sizeof(arr),file))
+    {
+        counter++;
+        if(counter == row)
+        {
+            fscanf(file,"%x , %x",&current,&reload);
+            break;
+
+        }
+
+    }
+ 
+ int subt = reload - current ;
+
+ return subt; 
+    
+    
+    
+
+}
+
+int writeReload(int reload)
```

Comment:
This functionality should happen in the ```BSP_Timer_Init()``` function. That function should also set some static global variable(s) that are function pointers.